rpm_test.go

```diff
@@ -31,3 +30,28 @@ func TestFileOwner(t *testing.T) {
 		t.Errorf("File owner shoud be %s but is %s", group, r.filegroups[0])
 	}
 }
+
+// https://github.com/google/rpmpack/issues/49
+func Test100644(t *testing.T) {
+	r, err := NewRPM(RPMMetaData{})
+	if err != nil {
+		t.Errorf("NewRPM returned error %v", err)
```

Comment:
Maybe just  t.Fatalf("NewRPM returned error %v", err) here?

Reply:
Thank you, Done.